UnityProject/Assets/Scripts/Util/StringExt.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Util
+{
+	internal static class StringExt
+	{
+		/// <summary>
+		/// Capitalizes the first letter in a string. Does not check for multiple words, only the first character in the string array.
+		/// </summary>
+		/// <param name="str">The string you want to manipulate.</param>
+		/// <returns>The same string with the first character in upper case.</returns>
+		public static string CapitalizeFirstLetter(this string str)
```

Comment:
I'm sure this already exists

Reply:
We have functions to capitalize the whole string, but not the first one.

Reply:
I'm sure I added that extension, also one that will capitalize every first letter in a sentence.

Reply:
Found it:
https://github.com/unitystation/unitystation/blob/3c52c81a360e411f89463f11236bb692041c5d3e/UnityProject/Assets/Scripts/Util/SweetExtensions.cs#L513

Feel free to move it if you wish, it is true it is very hard to find in that chaotical collection of extensions.